docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -17,24 +16,34 @@ You can view a list of in-progress and completed recoveries using the
 ==== Peer recovery settings
 
 `indices.recovery.max_bytes_per_sec` (<<cluster-update-settings,Dynamic>>)::
```

Comment:
Github won't let me suggest, but we may want to move the parenthetical inside the definition as with `indices.recovery.max_concurrent_file_chunks`.